src/Padawan/Parser/InlineDocBlockParser.php

```diff
@@ -10,7 +10,7 @@
 use PhpParser\Node\Expr\FuncCall;
 use Psr\Log\LoggerInterface;
 
-class InlineTypeHintParser {
+class InlineDocBlockParser {
```

Comment:
currently doc block is used for function and method arguments/return, class magic props and methods. Full list of possibilities is [here](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#77-global), so maybe it can return not only variables?